src/components/protocol_handler/test/protocol_payload_test.cc

```diff
@@ -75,7 +75,7 @@ void prepare_data(uint8_t* data_for_sending, ProtocolPayloadV2& message) {
   if (message.data.size() != 0) {
     uint8_t* current_pointer =
         data_for_sending + offset + message.json.length();
-    u_int32_t binarySize = message.data.size();
+    uint32_t binarySize = message.data.size();
```

Comment:
@BSolonenko const? `size_t` ?

Reply:
@AByzhynar fixed in https://github.com/Ford-Luxoft/sdl_core/pull/17/commits/5ea679815815e48385c34414d7eac980ecd337f3